src/components/table.js

```diff
@@ -227,6 +257,18 @@ const solveDate = (year,month)=>{
     }
     return year+"年"+(month+1)+"月";
 }
+
+function dashboard(text,index,t_month,region) {
+    if (index < 400 && region =='chinese' ) {
+        let [org_name, repo_name] = text.split("/");
+        let param = `{"org_name":"${org_name}","repo_name":"${repo_name}","t_month":"${t_month}"}`
+        param = window.btoa(param);
```

Comment:
<img width="1108" alt="image" src="https://user-images.githubusercontent.com/32434520/219549064-308d033c-a72e-4cfd-b637-14d8342026de.png">

This piece of code can be improved by `JSON.stringfy()`.

Reply:
What a great method, I never knew that before, thank you for the suggestion!